### ISSUE TITLE
Query demands required parameter after version update

### ISSUE THREAD

**Issue by QnavWhat** [nozzlegear/ShopifySharp#1078]
Hi,
I just updated from Version 6.3.0 to 6.6.0 and without changing the queries, I get the following error message when running GraphQL calls: 

_{"(400 Bad Request) query: Required parameter missing or invalid"}_

The same query runs fine with the previous version and I could find any change log with related information. The higher versions I tested return this message as well. I get the error message for every query I tested like ''_query MyQuery { shop { name } }_". When using e.g. POSTMAN I can run the query without exception.

Thanks and regards

**Comment by nozzlegear** [comment]
Hey @QnavWhat! Can you post the code you're using to send the query?

**Comment by QnavWhat** [comment]
Hi @nozzlegear 
this is a code snippet to reproduce 

`Public Shared Function GraphQL_Query() As String

    Dim ret As String = ""

    Try
        Dim loc = ShopifyHelper2024.serviceLocation.ListAsync().Result.Items.First
        Console.WriteLine(loc.Id)

        Dim body = "query { shop { id name } }"
        Dim res = ShopifyHelper2024.serviceGraphQL.PostAsync(body:=body).Result
        ret = res("shop")("name")
        Console.WriteLine(ret)

    Catch ex As Exception
        ret = ex.Message
    End Try

    Return ret

End Function`

The first call to serviceLocation is returning the location ID, which I just added to verify a correct initialisation of ShopifyHelper2024. Running the exact code with Version 6.4.0 will return Shop-Name but with Version 6.6.0 it will return error 400.

**Comment by QnavWhat** [comment]
Is there anything I can try t to adapt in my API-call?

**Comment by QnavWhat** [comment]
I tried using Version 6.18.0 to see the output. However a different error appeared. So I ran through the Version to test where related changes happend:

6.4.0 --> fine with the code posted above 
6.6.0-b118 --> **{"(400 Bad Request) query: Required parameter missing or invalid"}**
6.13.0-b182 --> **{"Auf das verworfene Objekt kann nicht zugegriffen werden." & vbCrLf & "Objektname: ""System.Net.Http.StringContent""."}**
6.18.0 --> still last error (which I guess relates to this issue https://github.com/nozzlegear/ShopifySharp/issues/1080)

**Comment by QnavWhat** [comment]
Hey @nozzlegear , ist my latest post somehow helpful?

**Comment by nozzlegear** [comment]
@QnavWhat it is helpful, yes! I'm sorry, I've been utterly swamped. I've set aside time to look at this and other ShopifySharp issues later this evening.

**Comment by nozzlegear** [comment]
Good news! I've found the issue, it's one of the overloads in the GraphService, which I have an open pull request for. I'm working on it right now, I'll publish it either tonight or tomorrow and will let you know when it's ready to test.

**Comment by QnavWhat** [comment]
Awesome, thanks a lot :)
Will test that asap after your publish

**Comment by QnavWhat** [comment]
Any news on this @nozzlegear :)

**Comment by nozzlegear** [comment]
@QnavWhat Yes, I've been actively working on that pull request I mentioned! I just need to finish up some unit tests and then it should be ready to merge and publish. I plan on doing that tomorrow/over the weekend.

Sorry it's taken so long!

**Comment by nozzlegear** [comment]
The PR is #1051.

**Comment by QnavWhat** [comment]
Ah cool, thank you for the information. Didn't meant to put preasure on that case ;)
